bin/kafka-server-start.sh

```diff
@@ -29,7 +29,7 @@ if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx1G -Xms1G"
 fi
 
-EXTRA_ARGS="-name kafkaServer -loggc"
+export EXTRA_ARGS=${EXTRA_ARGS-'-name kafkaServer -loggc'}
```

Comment:
Why was `export` added? The quotes also seem to be unnecessary, but I guess they don't hurt.